frontend/src/test/scala/bloop/bsp/BspCompileSpec.scala

```diff
@@ -603,7 +590,7 @@ class BspCompileSpec(
         assertSameExternalClassesDirs(fourthCompiledState, compiledState, projects)
         assertNoDiff(
           fourthCompiledState.lastDiagnostics(`A`),
-          """#4: task start 4
+          """#4: task start 3
```

Comment:
Any idea why these changed if there was no noop compilation beforehand?

Reply:
Yea exactly, since there was a no-op in there, there is no one less task which lowers the count.